Github.js

```diff
@@ -0,0 +1,151 @@
+{
+	"translatorID": "5e385e77-2f51-41b4-a29b-908e23d5d3e8",
```

Comment:
translatorID must be unique and you cannot reuse the one from your other translator https://github.com/zotero/translators/blob/bd311c6725863233f24c8cfd00ae842aa2a38e74/NCBI%20Nucleotide.js#L2

(This is similar to doi's which are unique;-)